kube-monitoring/README.md

```diff
@@ -55,85 +55,202 @@ After installation, Greenhouse will provide a generated link to the Prometheus u
+| kubeMonitoring.prometheus.plutonoDatasource | bool | `true` | Deploys a Plutono datasource config Secret for this Prometheus |
+| kubeMonitoring.prometheus.prometheusSpec.podMonitorSelector | object | `{"matchLabels":{"plugin":"{{ $.Release.Name }}"}}` | PodMonitors to be selected for target discovery. If {}, it will select all PodMonitors |
+| kubeMonitoring.prometheus.prometheusSpec.probeSelector | object | `{"matchLabels":{"plugin":"{{ $.Release.Name }}"}}` | Probes to be selected for target discovery. If {}, it will select all Probes |
+| kubeMonitoring.prometheus.prometheusSpec.ruleSelector | object | `{"matchLabels":{"plugin":"{{ $.Release.Name }}"}}` | PrometheusRules to be selected for target discovery. If {}, it will select all PrometheusRules |
+| kubeMonitoring.prometheus.prometheusSpec.scrapeConfigSelector | object | `{"matchLabels":{"plugin":"{{ $.Release.Name }}"}}` | scrapeConfigs to be selected for target discovery. If {}, it will select all scrapeConfigs |
+| kubeMonitoring.prometheus.prometheusSpec.serviceMonitorSelector | object | `{"matchLabels":{"plugin":"{{ $.Release.Name }}"}}` | ServiceMonitors to be selected for target discovery. If {}, it will select all ServiceMonitors |
+| kubeMonitoring.prometheus.prometheusSpec.storageSpec.volumeClaimTemplate.spec.resources.requests.storage | string | `"50Gi"` | How large the persistent volume should be to house the prometheus database. |
+| kubeMonitoring.prometheus.service.labels | object | `{"greenhouse.sap/expose":"true"}` | Add the label to expose the Prometheus service via Greenhouse. |
+
+### Service Discovery
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| kubeMonitoring.serviceDiscovery.pods.additionalMetricRelabelings | list | `[]` | MetricRelabelConfigs to apply to samples after scraping, but before ingestion. # ref: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#relabelconfig |
+| kubeMonitoring.serviceDiscovery.pods.additionalRelabelings | list | `[]` | RelabelConfigs to apply to samples before scraping ref: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#relabelconfig |
+| kubeMonitoring.serviceDiscovery.pods.enabled | bool | `true` | Enable pod discovery |
+| kubeMonitoring.serviceDiscovery.pods.jobLabel | string | `""` | Pod label for use in assembling a job name of the form <label value>-<port>  If no label is specified, the pod endpoint name is used. |
+| kubeMonitoring.serviceDiscovery.pods.limitToPrometheusTargets | bool | `true` | To avoid multiple pod scrapes from different Prometheis, service discovery can be limited to one target |
+| kubeMonitoring.serviceDiscovery.pods.namespaceSelector | object | `{"any":true}` | Namespaces from which pods are selected |
+| kubeMonitoring.serviceDiscovery.pods.port | string | `"metrics"` | Monitor Pods with the following port name |
+
+### Other Values
```

Comment:
can we ignore those ? and keep documented values to the minimum

Reply:
It is autogenerated. Do you know how to ignore these?

Reply:
bunch of # @ignored